src/Core/Validation/ConstraintValidator.php

```diff
@@ -17,8 +17,9 @@ class ConstraintValidator
     /**
      * Validate a value by a constraint
      *
+     * @param $value The value to validate
      * @param Constraint|Constraint[] $constraints a constraint or array of constraints to validate against
-     * @param string $fieldName The field name the value relates to, if relevant
+     * @param $fieldName The field name the value relates to, if relevant
```

Comment:
`@param` must always include the type. IDEs (and probably our API docs) don't know what to do with a non-typed param and they treat the param name as the type and the first word of the description as the name.

So `@param $value The value to validate` makes IDEs think `$value` is the type, and `The` is the name.

Either remove the `@param` entirely or revert the change. Personally I'd just remove the `$value` and `$fieldName` `@param`s as they're self-explanatory, and only keep the `$constraints` `@param`.

Reply:
This applies to any other PHPDoc changes in this PR.

Reply:
Will remove the `@params`